src/odemis/gui/util/img.py

```diff
@@ -1931,7 +1924,8 @@ def get_ordered_images(streams, raw=False):
         # merging without decreasing the intensity.
         if isinstance(ostream, acqstream.OpticalStream):
             images_opt.append((data, BLEND_SCREEN, ostream, md))
-        elif isinstance(ostream, (acqstream.SpectrumStream, acqstream.CLStream)):
+        elif isinstance(ostream, (acqstream.SpectrumStream, acqstream.CLStream)) and \
+            isinstance(s, RGBProjection):
```

Comment:
Which case is this handling? When is there a Spectrum or CL stream which is not a RGBProjection? Why is it a good idea to not show that image then? Please add a comment to explain.